dkg/pedersen/handler.go

```diff
@@ -92,31 +94,55 @@ func (s *state) SetParticipants(addrs []mino.Address) {
 	s.Unlock()
 }
 
+func (s *state) GetPublicKeys() []kyber.Point {
```

Comment:
General nitpick, but I think you don't need to change anything:

Adding public methods (`GetPublicKeys` and others) to a private struct (`state`) only makes sense if this private struct is returned somewhere out of the package. As it's only part of `Handler` as a private field, nobody outside of the package can access it.

So all these methods could be private, indicating to a reader of the source code that they don't need to take these methods into account if they're using it from the outside.

But as the other methods (`IsReshare` and others) were already here, I'd let it be.